packages/sitecore-jss-nextjs/src/editing/editing-healthz-middleware.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse, NextApiRequest } from 'next';
+
+/**
+ * Middleware / handler for use in the healthz Next.js API route (e.g. '/api/healthz')
+ * which is used by XMCloud container editing host.
+ */
+export class EditingHealthzMiddleware {
```

Comment:
Here maybe we can use "Healthcheck" (more generic) instead of "Healthz" (specific usage) and drop "Editing" (see other comment)
```suggestion
export class HealthcheckMiddleware {
```